sentences/en/_common.yaml

```diff
@@ -86,7 +86,7 @@ lists:
         out: "off"
   on_off_domains:
     values:
-      - in: light[s]
+      - in: (light|lamp)[s]
```

Comment:
The trouble with this is that when you say "turn off the bedroom lamps" without having a lamp, it will turn off LED strips or other stuff from the `light` domain, which is why I've refrained for adding anything other than `"light"` so far.

I'm not saying it's wrong to add it, but that's the end result. Do you think that's acceptable? cc @mrdarrengriffin

Reply:
Hmm. At first I was thinking that it world do no harm, the situation you describe I think is harmless because you have no other device you could be referring to. However, if there was a lamp or two in the room and instead it turned off everything, that could be unexpected. 

I’ll take that out.